certif/app/models/certification-candidate.js

```diff
@@ -53,6 +53,13 @@ export default class CertificationCandidate extends Model {
     return '-';
   }
 
+  get subscriptionScope() {
```

Comment:
Ce nom de fonction fait un raccourci dangereux niveau nommage car les souscriptions n'ont pas du tout une relation one-to-one avec un scope (millesime).

les scopes et millesimes c'est uniquement du v3. Les souscriptions sont compatibles avec les deux systemes de certif "vivantes" cote Pix App : certif v2 (et ses doubles souscriptions + ses badges) et v3 (et ses **simples** souscriptions et ses scopes + sa double certification CLEA )

Du coup je propose de bien rester sur le nom "souscription type" parce que sinon les futurs dev risauent de penser que souscription type = certif scope ce qui est pas vrai du tout.

```suggestion
  get subscriptionType() {
```